vue/components/ui/molecules/button-dropdown/button-dropdown.vue

```diff
@@ -193,6 +193,26 @@ export const ButtonDropdown = {
             };
             base["button-dropdown-" + this.size] = true;
             return base;
+        },
+        buttonColorPrimary() {
+            return this.primaryActive ? this.colorSecondary : this.colorPrimary;
+        },
+        buttonColorSecondary() {
+            return this.secondaryActive ? this.colorSecondary : this.colorPrimary;
+        },
+        styleButtonPrimary() {
+            const base = {};
+            base.color = this.primaryActive ? this.colorSecondary : this.colorPrimary;
+            base["background-color"] = this.primaryActive ? this.colorPrimary : this.colorSecondary;

```

Comment:
I would maintain the border-color property as well and would write, for consistency:

```js
base["color"] = ...
```

Reply:
Check border color with design

Reply:
Design team agreed the best approach is to always have the border to be this light grey color